bcda/bcdacli/cli.go

```diff
@@ -471,6 +471,17 @@ func setUpApp() *cli.App {
 				return err
 			},
 		},
+		{
+			Name:     "retrieve-suppression-hicn-to-bbid",
+			Category: "Data import",
```

Comment:
I updated BCDA-2562 (the ticket for `r43` deployment) to indicate that we'll need to trigger 1-800-Medicare ingestion as soon we deploy so that the BB ID is populated.